frontend/awx/resources/templates/hooks/useManageSurveyQuestions.tsx

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useManageItems } from '../../../../../framework/components/useManagedItems';
+import { useGet } from '../../../../common/crud/useGet';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { Spec, Survey } from '../../../interfaces/Survey';
+import { useParams } from 'react-router-dom';
+import { usePostRequest } from '../../../../common/crud/usePostRequest';
+
+export function useManageSurveyQuestions() {
+  const { t } = useTranslation();
+  const params = useParams<{ id: string }>();
+  const postRequest = usePostRequest();
+  const endpoint = awxAPI`/job_templates/${params.id?.toString() ?? ''}/survey_spec/`;
+  const columns = useMemo(
+    () => [
+      {
+        header: t('Survey questions'),
+        cell: (spec: Spec) => spec.question_name,
+      },
```

Comment:
Could we add a second column, "Defaults"  or "Default Answer(s)", to keep feature parity with the old ui?